### ISSUE TITLE
WhatsApp Web Version string should be user-editable

### ISSUE THREAD

**Issue by rvdborgt** [hoehermann/purple-gowhatsapp#81]
Pidgin suddenly says:

"restoring failed: error decoding login connResp: json: cannot unmarshal array into Go value of type map[string]interface {}"

... and remains disabled.
Googling this message finds these two 2-year-old closed issues:
https://github.com/dimaskiddo/go-whatsapp-rest/issues/6
https://github.com/Rhymen/go-whatsapp/issues/170

Edit: I saw there was a new build from 13 June but that didn't fix the problem. I'm on Windows 10.

**Comment by hoehermann** [comment]
Confirmed. I guess, this is a problem within go-whatsapp rather than this plug-in, but I am not sure.

**Comment by verm** [comment]
Yes see: https://github.com/Rhymen/go-whatsapp/issues/571  It's been fixed there already.

**Comment by hoehermann** [comment]
@verm Nice to know. The nightly builds will be updated automatically.

@pkar70 I'll put this on the list of proposed enhancements.

**Comment by rvdborgt** [comment]
Today's nightly build solves the issue.

**Comment by twinklebob** [comment]
Are there still nightly builds for ubuntu 18? I'm not seeing one for last night. I'm assuming the ubuntu 20 one probably won't work.

**Comment by pkar70** [comment]
Nightly build... but what about Windows version? :)

**Comment by hoehermann** [comment]
@pkar70 Nightly builds for Windows are available at the [build-bot](https://buildbot.hehoe.de/purple-gowhatsapp/builds/?C=M;O=D). It seems quite reliable. So far, I only broke it once. :D

@twinklebob I do not know. golang builds use static linking and the libpurple API did not change. Feel free to give the 20.04 build a try on 18.04. Could work.

@rvdborgt Nice to know. Thank you. :)

**Comment by twinklebob** [comment]
This one has reared it's ugly head again, can someone do the necessary stuff and trigger a new nightly build? I forgot to update last time, but the ubuntu 20.04 version worked fine for me.

**Comment by rvdborgt** [comment]
Did you try the most recent nightly build, from after #93 was fixed? The Windows version works for me.

**Comment by twinklebob** [comment]
:facepalm: i just skimmed the list and saw loads of `0.4.1` and didn't even notice the recent date. What a numpty, and a rude one at that, because I also hadn't checked the other tickets. Ignore me, I'll go back to my day job, complaining about users leaving reports like I just left. I'm sure the update will work.

**Comment by hoehermann** [comment]
This is no longer relevant with the whatsmeow branch.
